test/combineValidators.test.js

```diff
@@ -4,25 +4,29 @@ import combineValidators from '../src/combineValidators';
 import isRequired from '../src/validators/isRequired';
 import isAlphabetic from '../src/validators/isAlphabetic';
 import isNumeric from '../src/validators/isNumeric';
+import matchesValue from '../src/validators/matchesValue';
 
 const validatePerson = combineValidators({
   name: composeValidators(
     isRequired,
     isAlphabetic
   )('Name'),
 
+  confirmName: matchesValue('name', 'Name')('Confirm Your Name'),
```

Comment:
The second argument `'Name'` will technically be optional since you're providing a custom message. Also the custom message syntax is wrong here. The string `'Confirm Your Name'` will be treated as the field name. You'll want to change it to `{ message: 'Confirm Your Name' }`.